index.js

```diff
@@ -144,6 +144,86 @@ function reset () {
   moduleAliasNames = []
 }
 
+//
+// Utility function
+//
+
+// Removes file name from the base and returns only dir.
+function getBaseDir (base) {
+  var parsed = nodePath.parse(base)
+  return parsed.ext ? parsed.dir : base
+}
+
+//
+// Functions to load configurations from a file
+//
+
+// Load configuration from any file
+function loadConfigFile (configPath) {
+  try {
+    var config = require(configPath)
+
+    // ES6 export default
+    if (config && config.default) {
+      return config.default
+    }
+
+    return config
+  } catch (e) {
+    // Do nothing
+  }
+}
+
+// Load configuration from the package.json file
+function loadPackageJSONFile (configPath) {
+  try {
+    var config = require(configPath)
+
+    if (config && config['module-alias']) {
+      return config['module-alias']
+    }
+
+    return {}
+  } catch (e) {
+    // Do nothing
+  }
+}
```

Comment:
We could merge both functions, and look for `config.default` first, and then config['module-alias']